docs/topics/Opcodes_and_Status_Codes.md

```diff
@@ -137,6 +137,7 @@ Along with the HTTP error code, our API can also return more detailed error code
 | 10062  | Unknown interaction                                                                                                           |
 | 10063  | Unknown application command                                                                                                   |
 | 10066  | Unknown application command permissions                                                                                       |
+| 10067  | Unknown Stage Instance                                                                                                        |
```

Comment:
shouldn't it be like this `Unknown stage instance`?

Reply:
No, that's correct
![image](https://user-images.githubusercontent.com/24302436/122725346-cb0f2380-d26c-11eb-958c-27713753775b.png)